test/src/org/labkey/test/util/cds/CDSHelper.java

```diff
@@ -1372,35 +1353,23 @@ public void closeInfoPage()
     @LogMethod (quiet = true)
     public void deleteGroupFromSummaryPage(@LoggedParam String name)
     {
-        BaseWebDriverTest.sleep(500);
-        goToLearnGroupsPage();
-        goToGroupFromLearnGrid(name);
-        _test.waitForElement(Locators.cdsButtonLocator("Delete"));
-        Locators.cdsButtonLocator("Delete").findElement(_test.getWrappedDriver()).click();
-        _test.waitForText("Are you sure you want to delete");
-        Locators.cdsButtonLocator("Delete", "x-toolbar-item").notHidden().findElement(_test.getWrappedDriver()).click();
-        _test.waitForText(HOME_PAGE_HEADER);
-        goToLearnGroupsPage();
+        _test.beginAt("/cds/" + _test.getPrimaryTestProject() + "/cds-app.view#learn/learn/Group", WebDriverWrapper.WAIT_FOR_PAGE);
```

Comment:
This url isn't quite right. It specifies the `cds` controller twice. (Note: the server seems to handle this just fine)
```suggestion
        _test.beginAt(_test.getPrimaryTestProject() + "/cds-app.view#learn/learn/Group", WebDriverWrapper.WAIT_FOR_PAGE);
```
You could also use a URL builder
```suggestion
        _test.beginAt(WebTestHelper.buildUrl("cds", _test.getPrimaryTestProject(), "app.view#learn/learn/Group"), WebDriverWrapper.WAIT_FOR_PAGE);
```

Reply:
Done